web/Views/Landlord/Properties.cshtml

```diff
@@ -1,79 +1,33 @@
+@using BricksAndHearts.Auth
 @model PropertiesDashboardViewModel
 
 @{
-    ViewData["Title"] = "My Properties";
+    ViewData["Title"] = "Properties";
 }
 
-@if (Model.Owner == null)
-{
-    <h2>Landlord's properties</h2>
-}
-else
-{
-    <h2>
-        @Model.Owner.FirstName's properties
-    </h2>
-}
+<div class="mx-md-5">
+    <h1 class="font-h1 mt-5">
+        @if (Model.Owner == null)
+        {
+            @("Landlord's properties")
+        }
+        else if (Model.Owner.LandlordId == ((BricksAndHeartsUser)User.Identity!).LandlordId)
+        {
+            @("My properties")
+        }
+        else
+        {
+            @(Model.Owner.FirstName + "'s properties")
+        }
+    </h1>
+
+    <div class="row mt-4">
+        @* Sorting and filtering goes here *@

```

Comment:
Does this have a ticket number? :)

Reply:
I don't think there's a ticket for it at the moment, I just left a space for it in case someone adds in the future.